server/routes/deleteObject.js

```diff
@@ -33,27 +34,29 @@ router.post('/:url', async function (req, res) {
     let objects = [];
     let stream = minioClient.listObjects(
       bucketName,
-      'hv/' + user + '/' + name + fileId,
+      'hv/' + user + '/' + name,
       true,
     );
     stream.on('data', async obj => {
       objects.push(obj.name);
```

Comment:
The error handling for removing objects should also log the error using the `log` object instead of `console.error`. This maintains consistency in logging.
*This comment was auto-generated by **Code Bot** to provide suggestions and improvements.*